src/components/ApplnBar.js

```diff
@@ -151,8 +230,104 @@ const ApplnBar = () => {
           <Typography variant='h6' className={classes.title}>
             Agri-Man
           </Typography>
-          <Button color='inherit'>Login</Button>
+          {loginVisible ? (
```

Comment:
Please give it a shot to see if this whole thing could be a separate component to keep this file cleaner.

Reply:
will do, in next PR